dataretrieval/nwis.py

```diff
@@ -24,9 +24,12 @@
 WATERDATA_SERVICES = ['qwdata', 'measurements', 'peaks', 'pmcodes', 'water_use', 'ratings']
 
 
-def format_response(df, service=None):
+def format_response(df, multi_index=None, service=None):
```

Comment:
Parameter order for functions matter, so it is often best to put new parameters last.

Reply:
Yes, service is only used in 1 function, so it is more convinient to add it as second. If third, every call to multi_index will need to be named as service=None most of the times. Do you think it is still better to add multi_index last?